### PR TITLE
Assignment-4th(CountsManager) Push

### PR DESCRIPTION
**Pull request by vinayakfexle**
In this Assignment we are writing a Class to demonstrate two methods which will update number of Classes and number of student fields on school object.

### CODE REVIEW

File: src/classes/CountsManager.cls [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+*  Revision Logs    :   V_1.0 - Created
+*
+**/
+
+
+public without sharing class CountsManager
+{
+    public static void updateNoOfStudentsDetails(String schoolName)
+    {
+        List<School__c> schoolToBeUpdated = [SELECT Id, Name
```
Comment (SeemuSaikiaSF) [inline src/classes/CountsManager.cls:20]: Use LIMIT 1 in the  SOQL Query, as you are expecting only 1 School record for the schoolName .

File: src/classes/CountsManager.cls [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                                       FROM School__c 
+                                       where School__c.Name = :schoolName];
```
Comment (SeemuSaikiaSF) [inline src/classes/CountsManager.cls]: Use LIMIT 1 in the SOQL Query, as you are expecting only 1 School record for the schoolName .

File: src/classes/CountsManager.cls [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            AggregateResult studentCount = [SELECT Count(Student__c.Id)
+                				   FROM Student__c 
+                				   WHERE School__r.Id = :schoolToBeUpdated[0].Id];
+           
+            double countOfStudent = (double) studentCount.get('expr0');
+            schoolToBeUpdated[0].No_Of_Students__c = countOfStudent;
+            Update schoolToBeUpdated[0];
+            System.debug(schoolToBeUpdated[0].Id);
+        }
+        Else
```
Comment (SeemuSaikiaSF) [inline src/classes/CountsManager.cls]: else

File: src/classes/CountsManager.cls [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            
+        }
+    }
+            
+    
+    public static void updateNoOfClassesDetails(String schoolId)
+    {
+        List<School__c> schoolToBeUpdated = [SELECT Id, Name
+                                       FROM School__c 
+                                       where School__c.Id = :(Id)schoolId];
```
Comment (SeemuSaikiaSF) [inline src/classes/CountsManager.cls]: Use LIMIT 1

**Comment by vinayakfexle** [comment]
Closing CountsManager.cls PR To create a new PR for Updated CountsManager.cls

**Pull request closed without merging**